src/page/MemberInfo/index.tsx

```diff
@@ -26,50 +22,7 @@ export default function MemberInfo() {
       <div css={S.contentContainer}>
         <div css={S.topBar} />
         <div css={S.buttonContainer}>
-          <ToggleButtonGroup
-            color="primary"
-            value={track}
-            exclusive
-            onChange={handleTrackChange}
-            aria-label="Platform"
-          >
-            <ToggleButton value="all" sx={{ width: '85px' }}>
-              ALL
-            </ToggleButton>
-            <ToggleButton
-              value="frontend"
-              sx={{ textTransform: 'none', width: '85px' }}
-            >
-              FrontEnd
-            </ToggleButton>
-            <ToggleButton
-              value="backend"
-              sx={{ textTransform: 'none', width: '85px' }}
-            >
-              BackEnd
-            </ToggleButton>
-            <ToggleButton
-              value="android"
-              sx={{ textTransform: 'none', width: '85px' }}
-            >
-              Android
-            </ToggleButton>
-            <ToggleButton
-              value="ios"
-              sx={{ textTransform: 'none', width: '85px' }}
-            >
-              iOS
-            </ToggleButton>
-            <ToggleButton value="uiux" sx={{ width: '85px' }}>
-              UI/UX
-            </ToggleButton>
-            <ToggleButton
-              value="etc"
-              sx={{ textTransform: 'none', width: '85px' }}
-            >
-              etc
-            </ToggleButton>
-          </ToggleButtonGroup>
+          <TrackFilter />
```

Comment:
좀 더 좁은단위의 Suspense가 필요해보여요~ 목록도 그렇구
```suggestion
          <Suspense>
            <TrackFilter />
          </Suspense>
```

Reply:
suspense query를 사용한 컴포넌트만 Suspense로 감싸도록 수정했습니다. 67a9a0f4cc1bcd03b6c256806381954d7598d6d3